cmd/crc/cmd/start.go

```diff
@@ -63,19 +66,73 @@ func runStart(arguments []string) error {
 
 	client := machine.NewClient()
 	result, err := client.Start(startConfig)
+	return render(&startResult{
+		Success:       err == nil,
+		Error:         errorMessage(err),
+		ClusterConfig: toClusterConfig(result),
+	}, os.Stdout, outputFormat)
+}
+
+func toClusterConfig(result machine.StartResult) *clusterConfig {
+	if result.Error != "" {
+		return nil
+	}
+	return &clusterConfig{
+		URL: result.ClusterConfig.ClusterAPI,
+		AdminCredentials: credentials{
+			Username: "kubeadmin",
+			Password: result.ClusterConfig.KubeAdminPass,
+		},
+		DeveloperCredentials: credentials{
+			Username: "developer",
+			Password: "developer",
+		},
+	}
+}
+
+func errorMessage(err error) string {
 	if err != nil {
-		return err
+		return err.Error()
 	}
-	logging.Warn("The cluster might report a degraded or error state. This is expected since several operators have been disabled to lower the resource usage. For more information, please consult the documentation")
+	return ""
+}
 
-	output.Outln("Started the OpenShift cluster.")
-	output.Outln("")
-	output.Outln("To access the cluster, first set up your environment by following 'crc oc-env' instructions.")
-	output.Outf("Then you can access it by running 'oc login -u developer -p developer %s'.\n", result.ClusterConfig.ClusterAPI)
-	output.Outf("To login as an admin, run 'oc login -u kubeadmin -p %s %s'.\n", result.ClusterConfig.KubeAdminPass, result.ClusterConfig.ClusterAPI)
-	output.Outln("")
-	output.Outln("You can now run 'crc console' and use these credentials to access the OpenShift web console.")
-	return nil
+type clusterConfig struct {
+	URL                  string      `json:"url"`
+	AdminCredentials     credentials `json:"adminCredentials"`
+	DeveloperCredentials credentials `json:"developerCredentials"`
+}
+
+type credentials struct {
+	Username string `json:"username"`
+	Password string `json:"password"`
+}
+
+type startResult struct {
+	Success       bool           `json:"success"`
+	Error         string         `json:"error,omitempty"`
+	ClusterConfig *clusterConfig `json:"clusterConfig,omitempty"`
+}
```

Comment:
Without having looked at these changes in detail, I would expect the daemon API json and the CLI json to be the same, the daemon API would be similarly impacted by fields renamed by mistake

Reply:
From what I understand, daemon API is private and can break at any time. With these new types, I wanted to not force us to change right now the 2 trays. Tray work is still experimental so, I tought it would be good if we don't link this 2 stories.

If the daemon API is meant to be public, then I think we need to add an API version somewhere (in every object like k8s?).

Reply:
> From what I understand, daemon API is private and can break at any time. With these new types, I wanted to not force us to change right now the 2 trays. Tray work is still experimental so, I tought it would be good if we don't link this 2 stories.

Ah in that case I'd be fine with an issue reminding us the daemon and the cli should use the same json output (I'm assuming that's the case now?)